### ISSUE TITLE
Why is the SqlDataReader not disposed in this example?

### ISSUE THREAD

**Issue by FrancescoBonizzi** [dotnet/docs#5914]
Why is the SqlDataReader not disposed in this example? I thought that it should be enclosed in a using statement or disposed after using it.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a3a332d5-d37a-aab8-880f-a9d3d3cee053
* Version Independent ID: de5755fd-9dc0-a7fb-627f-2992ac560ec7
* Content: [SqlDataReader.Close Method (System.Data.SqlClient)](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqldatareader.close?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(System.Data.SqlClient.SqlDataReader.Close);k(TargetFrameworkMoniker-.NETFramework,Version%3Dv4.5.2);k(DevLang-csharp)%26rd%3Dtrue&view=netframework-4.7.2)
* Content Source: [xml/System.Data.SqlClient/SqlDataReader.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Data.SqlClient/SqlDataReader.xml)
* Product: **.net**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

**Comment by FrancescoBonizzi** [comment]
(If needed I can pull an edit)

**Comment by mairaw** [comment]
Thanks @FrancescoBonizzi. @douglaslMS can you look at this issue?

**Comment by mjwills** [comment]
Does `reader` really need to be `Close`d **and** `Dispose`d, as shown in this sample?

**Comment by douglaslMS** [comment]
@mjwills You're correct that Dispose calls Close (although the inverse is not true - Close does not call Dispose).
